ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/link/v1/link/link.html

```diff
@@ -0,0 +1,32 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div class="cmp-link">
+    <sly data-sly-use.commonsTemplates="core/wcm/components/commons/v1/templates.html"
+         data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+         data-sly-use.link="com.adobe.cq.forms.core.components.models.formsportal.Link"
+         data-sly-test.hasText="${link.title}">
+        <a
+            id="${component.id}"
```

Comment:
Why do you need component, you could have link.id here ?

Reply:
Indeed link.id would work
But I'm not sure why it was done in the core-components repository
I saw this pattern used in several components (e.g [Download component](https://github.com/adobe/aem-core-wcm-components/blob/master/content/src/content/jcr_root/apps/core/wcm/components/download/v1/download/download.html#L20), [Embed component](https://github.com/adobe/aem-core-wcm-components/blob/master/content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embed.html#L20) and many others), so I just used it



Should I change this to link.id ?

Reply:
Changed to link.id